docs/tutorials/typescript/04-validation.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_label: Validation logic
+sidebar_position: 040
+title: Writing the validation logic
+description: A tutorial on writing validation logic for a Kubewarden policy using TypeScript.
+keywords: [kubewarden, kubernetes, writing policies, typescript]
+doc-type: [tutorial]
+doc-topic: [kubewarden, writing-policies, typescript, validation-logic]
+doc-persona: [kubewarden-policy-developer]
+---
+
+<head>
+  <link rel="canonical" href="https://docs.kubewarden.io/tutorials/writing-policies/typescript/validation"/>
+</head>
+
```

Comment:
We should put the very same warning box we put on the policy settings page, the one about not writing to STDOUT